cli/impl/src/main/java/io/helidon/build/cli/impl/InitCommand.java

```diff
@@ -93,6 +98,17 @@ protected void invoke(CommandContext context) throws Exception {
             assertSupportedVersion(helidonVersion);
         }
 
+        // Set system properties from a file if provided on command-line
+        if (initOptions.propsFileOption() != null) {
+            try (InputStream input = new FileInputStream(initOptions.propsFileOption())) {
```

Comment:
Maybe create a utility method in FileUtils to load properties from a file. E.g. `FileUtils.loadProperties(Path)`